src/main/java/seedu/address/ui/CommandBox.java

```diff
@@ -17,18 +17,37 @@ public class CommandBox extends UiPart<Region> {
     private static final String FXML = "CommandBox.fxml";
 
     private final CommandExecutor commandExecutor;
+    private final Suggestions suggestions;
 
     @FXML
     private TextField commandTextField;
```

Comment:
could maybe add some basic validation for the commandTextField input to catch unsupported characters early